packages/browser/src/tracing/request.ts

```diff
@@ -310,17 +312,21 @@ export function xhrCallback(
 
   const hasParent = !!getActiveSpan();
 
+  const parsedUrl = parseUrl(sentryXhrData.url);
+
   const span =
     shouldCreateSpanResult && hasParent
       ? startInactiveSpan({
           name: `${sentryXhrData.method} ${sentryXhrData.url}`,
           attributes: {
             type: 'xhr',
             'http.method': sentryXhrData.method,
+            'http.url': sentryXhrData.url,
             url: sentryXhrData.url,
+            'server.address': parsedUrl.host,
```

Comment:
Naive question, but will relative URLs have a host? I'm not sure whether `sentryXhrData.url` has a domain or not, and for my use-case I need `server.address` for relative URLs. I didn't see a spec for it, so I wasn't sure how it works

Reply:
Hmm, I'll add a test for this as well! Let's see 😅